### DISCUSSION TITLE
license term

### DISCUSSION THREAD

**Discussion by 77777773** [explosion/spaCy#11581] [Help: Other Questions] [answered]
Hi,

Organizations havent membership agreement with LDC for OntoNotes can use spacy models for commercial purposes freely?

**Comment by rmitsch** [comment, marked as answer]
Hi @77777773! Have a look at [this thread](https://github.com/explosion/spaCy/issues/5394#issuecomment-623330210) - in general, always check out the metadata included in the model files.

For models derived from OntoNotes, like English: 
> We have a membership agreement with LDC for OntoNotes that allows us to distribute models with an MIT license, so this should not be a problem. (OntoNotes is available under several different licenses, which depend on the organization's membership with LDC. The for-profit membership agreement allows for commercial use of derived work.)
